xds/src/test/java/io/grpc/xds/BootstrapperImplTest.java

```diff
@@ -586,17 +583,7 @@ public void useV2ProtocolByDefault() throws XdsInitializationException {
   }
 
   @Test
-  public void supportV3Protocol_disabledByDefault() throws XdsInitializationException {
-    subtestSupportV3Protocol(false);
-  }
-
-  @Test
-  public void supportV3Protocol_enabled() throws XdsInitializationException {
-    BootstrapperImpl.enableV3Protocol = true;
-    subtestSupportV3Protocol(true);
-  }
-
-  private void subtestSupportV3Protocol(boolean enabled) throws XdsInitializationException {
+  public void useV3ProtocolIfV3FeaturePresent() throws XdsInitializationException {
```

Comment:
nit: maybe just `supportV3Protocol()`? Whatever...

Reply:
Yeah, then the other test should be renamed to `supportV2Protocol()`. I'll keep it as-is.